### PR TITLE
ENH: Increase code coverage

### PR DESCRIPTION
**Pull request by jhlegarreta**
Increase the code coverage:
- Exercise the basic object methods using the
  `ITK_EXERCISE_BASIC_OBJECT_METHODS` macro.
- Remove explicit calls to the `Print` method and rely on the
  basic method exercising macro call.

Take advantage of the commit to:
- Define as a constant expression the image dimensionality.
- Capitalize comment sentences.
in the `itkHausdorffDistanceImageFilterTest.cxx` file

## PR Checklist
- [X] No [API changes](https://github.com/InsightSoftwareConsortium/ITK/blob/master/CONTRIBUTING.md#breaking-changes) were made (or the changes have been approved)
- [X] No [major design changes](https://github.com/InsightSoftwareConsortium/ITK/blob/master/CONTRIBUTING.md#design-changes) were made (or the changes have been approved)

### CODE REVIEW

File: Modules/Filtering/DistanceMap/test/itkHausdorffDistanceImageFilterTest.cxx [hunk 1]
```diff
@@ -92,7 +93,7 @@ itkHausdorffDistanceImageFilterTest(int, char *[])
     filter->Update();
     filter->Print(std::cout);
```
Comment (dzenanz) [inline Modules/Filtering/DistanceMap/test/itkHausdorffDistanceImageFilterTest.cxx:94]: Is this print left on purpose?

**Review by dzenanz — APPROVED** [review]

**Review by thewtex — APPROVED** [review]
Thanks @jhlegarreta !

File: Modules/Filtering/DistanceMap/test/itkHausdorffDistanceImageFilterTest.cxx (hunk 1 above)
Comment (jhlegarreta) [reply Modules/Filtering/DistanceMap/test/itkHausdorffDistanceImageFilterTest.cxx:94]: Good catch. Will address that later tonight.

Comment (jhlegarreta) [reply Modules/Filtering/DistanceMap/test/itkHausdorffDistanceImageFilterTest.cxx]: Done in 3b36da7.

**Review by dzenanz — APPROVED** [review]

**Review by hjmjohnson — APPROVED** [review]

**Pull request merged**